### PR TITLE
Hidden problem

### PR DESCRIPTION
**Pull request by timothydm42**
global feature: `hidden` global doesn't get overridden

Adds `!important` when `hidden` sets `display:none` 

Breaks this rule:
```Use only tag selectors to style an element. You may only use pseudo selectors (such as :before, :hover) and attribute selectors when used in conjunction with a tag selector (i.e. use input[type=text] instead of just [type=text]).```

additionally; do we want to be overriding browser behavior; is this sustainable? A long term good idea?

### CODE REVIEW

**Comment by straker** [comment]
Ah the lovely hidden attribute problem. This is the 2nd (or 3rd?) global selector that doesn't fit into one of the already defined files. Maybe we should create a `_globals.styl` or `_overrides.styl` file that includes the `sr-only, visually-hidden` rule and this one?

**Comment by timothydm42** [comment]
something like this?

File: assets/css/helpers/_overriddes.styl [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/*
```
Comment (straker) [inline assets/css/helpers/_overriddes.styl]: All stylus files import the entire helpers dir `@import '/components/fs-styles/assets/css/helpers/*`, so these classes will end up in everyones individual style sheets. This file should probably go in the Elements dir even though it's not an element per say

Comment (timothydm42) [reply assets/css/helpers/_overriddes.styl]: Thanks!

File: README.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ To help organize a large CSS code base, we have split the files into different f
 
 Folder     | Contents
 -----------|---------
-Helpers    | Variables, mixins, and functions.
+Helpers    | Variables, mixins, overrides, and functions.
```
Comment (straker) [inline README.md]: You'll need to change this description as well over to Elements (here and in the README of helpers)

Comment (timothydm42) [reply README.md]: indeed

**Comment by timothydm42** [comment]
Thanks gentlemen, will do @liechtym

**Comment by joeycozza** [comment]
"This man is a genius!", I opined!

**Pull request merged**